### ISSUE TITLE
cloudflare_zone import issue - zone not set

### ISSUE THREAD

**Issue by abohne** [terraform-providers/terraform-provider-cloudflare#169]
importing a zone doesn't set zone forcing a new resource next plan/apply.

terraform import cloudflare_zone.myzone ba6556bfef261f3373d1d544af786ccb
Import successful!

on next plan
zone: "" => "domain.name" (forces new resource)

Similar to #161 and #165

provider.cloudflare 1.9

**Comment by chasen** [comment]
I too am having this issue, is there any known workaround?

**Comment by chasen** [comment]
Found the answer to my own question:

After you import the zone, you can go into your tfstate file and manually update the zone:

```
"cloudflare_zone.domainName": {
    "type": "cloudflare_zone",
    "depends_on": [],
    "primary": {
        "id": "<ID>",
        "attributes": {
            "id": "<ID>",
            "meta.%": "2",
            "meta.phishing_detected": "false",
            "meta.wildcard_proxiable": "false",
            "name_servers.#": "2",
            "name_servers.0": "mira.ns.cloudflare.com",
            "name_servers.1": "pete.ns.cloudflare.com",
            "paused": "false",
            "status": "active",
            "type": "full",
            "vanity_name_servers.#": "0",
            "zone": "domainName.com" // <-- Add this line
        },
        "meta": {},
        "tainted": false
    },
    "deposed": [],
    "provider": "provider.cloudflare"
},
```

**Comment by jacobbednarz** [comment]
`cloudflare_zone` doesn't have a provider specific import and instead just relies on [`ImportStatePassthrough`](https://github.com/hashicorp/terraform/blob/af9d046afbafd92a7d74d301da634b8bc1eb7bfa/helper/schema/resource_importer.go#L47-L52) which essentially just passes in the ID and then the code makes some assumptions about what it is. 

I'll submit a PR to add an `Import` method to allow setting this instead of relying on the assumptions.

@chasen I don't recommend munging state files by hand as it can often leads to more issues.

**Comment by patryk** [comment]
Fixed in `1.10`.
